lib/shoryuken/manager.rb

```diff
@@ -173,11 +173,10 @@ def next_queue
       # get/remove the first queue in the list
       queue = @queues.shift
 
-
-      unless Shoryuken.workers.include? queue
+      unless defined?(::ActiveJob) || Shoryuken.workers.include?(queue)
```

Comment:
@elsurudo 

> Queue pausing/sleeping is disabled when ActiveJob is present. This is needed because the workers are dynamically-defined, so the queues would be paused indefinitely and jobs wouldn't be processed.

After we dynamically add a worker using `register_worker!`, `Shoryuken.workers.include? queue` will return `true`, then it should stop constantly pausing. The cons would be the huge number of debug message until the worker is registered.

Just wondering because if there's no worker defined yet, the processor will fail, because of no workers.

https://github.com/elsurudo/shoryuken/blob/active-job/lib/shoryuken/processor.rb#L13

wdyt?